WS17_18/01_introduction/your assignment/Yatodoa/ViewController.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  ViewController.swift
+//  Yatodoa
+//
+//  Created by Armin Galinowski on 24.10.17.
+//  Copyright © 2017 Armin Galinowski. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    func getTimestamp() -> String{
+        let now = Date()
+        
+        let uhrzeit = DateFormatter()
+        uhrzeit.timeZone = TimeZone.current
+        uhrzeit.dateFormat = "HH:mm"
+        
+        let tag = DateFormatter()
+        tag.timeZone = TimeZone.current
+        tag.dateFormat="dd.MM.yyyy"
+        
+        
+        let datum = tag.string(from: now)
+        let uhr = uhrzeit.string(from: now)
+        
+        return "erledigt am: " + datum + " um " + uhr + " Uhr"
+    }
+
+    
+    
+    
+    var todos: [String]=[]{
+        didSet{
+            print(todos)
+            
+            if(todos.count==0){
+                statusText.text = "keine Aufgaben erledigt"
+                             }
+            else if(todos.count==1){
+                statusText.text = "1 Aufgabe erledigt"
+
+            }
+            else{
+                statusText.text = "\(todos.count) Aufgaben erledigt"
+            }
+            
+        }
+    }
+    
+    
+    @IBOutlet weak var statusText: UILabel!
+    @IBOutlet weak var stamp_1: UILabel!
+    @IBOutlet weak var stamp_2: UILabel!
+    
+    
+    @IBOutlet weak var todotextField2: UITextField!
+    
+    @IBAction func checkboxTapped_2(_ sender: UIButton) {
```

Comment:
Hier hätte man die Implementierung von `checkoxTapped` und `checkboxTapped_2` in eine [Funktion auslagern können](https://github.com/alexdobry/FSIOS/pull/7), damit man keine redundanten Code hat.